iroha_client_cli/tests/test_config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "TORII_CONFIGURATION": {
+    "TORII_URL": "127.0.0.1:1337",
+    "TORII_CONNECT_URL": "127.0.0.1:8888"
+  },
+  "SUMERAGI_CONFIGURATION": {
+    "TRUSTED_PEERS": [
+      {
+        "address": "127.0.0.1:1337",
+        "public_key": {"inner": [101, 170, 80, 164, 103, 38, 73, 61, 223, 133, 83, 139, 247, 77, 176, 84, 117, 15, 22, 28, 155, 125,     80, 226, 40, 26, 61, 248, 40, 159, 58, 53]}
+      }
+    ]
+  },
+  "KURA_CONFIGURATION": {
+    "KURA_INIT_MODE": "strict",
+    "KURA_BLOCK_STORE_PATH": "./blocks"
+  },
+  "BLOCK_SYNC_CONFIGURATION": {
+    "GOSSIP_PERIOD_MS": 10000,
+    "BATCH_SIZE": 1
+  },
+  "QUEUE_CONFIGURATION" : {
+    "MAXIMUM_TRANSACTIONS_IN_BLOCK": 2
+  },
```

Comment:
If it is iroha config, then it should also contain `LOGGER_CONFIGURATION` I think.